pkg/agent/agent_test.go

```diff
@@ -107,6 +107,23 @@ func TestInitstore(t *testing.T) {
 	if !found2 {
 		t.Errorf("Failed to load OVS port into local store")
 	}
+
+	// OVS port external_ids should be updated to set AntreaInterfaceTypeKey if it is not existing in OVSPortData.
```

Comment:
Is "it is not existing" correct? I think it should be "it doesn't exist". @antoninbas

Reply:
updated.